packages/q/qadwaitadecorations/package.yml

```diff
@@ -0,0 +1,41 @@
+# NOTE TO PACKAGERS: rebuild this after every Qt stack upgrade due to using private headers!
```

Comment:
This note isn't necessary. Instead add it to the list of rebuilds in the qt5-base repo.